pennylane/ops/op_math/pow.py

```diff
@@ -267,6 +274,22 @@ def compute_sparse_matrix(*params, base=None, z=0):
             return base_matrix**z
         raise SparseMatrixUndefinedError
 
+    def sparse_matrix(self, wire_order=None):
+        try:
+            pr = self._pauli_rep
+            wires = wire_order or self.wires.tolist()
+            return pr.to_mat(wire_order=wires, format="csr")
+        except NotImplementedError:
```

Comment:
Important to remember that exceptions should be exceptional.  `except` logic can be expensive, so the error should only occur a small percentage of the time.

Reply:
Removed the try except! Using None instead of excepting error made the code much cleaner